### PR TITLE
SideBar fix #296

### PR DESCRIPTION
**Pull request by Marc13000**
Fixes polypheny/Polypheny-DB#296

### CODE REVIEW

**Comment by datomo** [comment]
image
_Your Change_

This fix is sadly a bit too radical, while it removes the extending and minimizing buttons, it does so for every page and additionally removes the schema tree and the configs/settings in the process.
image
image
_Your Change_


These are both necessary for Polypheny to work correctly.
To fix this issue you would need to only remove the minimizing and extending buttons for the **monitoring**, **config**, **uml**, **query console** and **plan builder view**  pages, like mentioned in https://github.com/polypheny/Polypheny-DB/issues/296#issuecomment-800089260. This could be done by for example checking the router for the used route and applying an adapted form of your proposal only for those cases. In this scenario you would also need to make sure that the normal buttons are still visible.

**Comment by Marc13000** [comment]
Got it thanks for the help!

**Comment by Marc13000** [comment]
@datomo I fixed it as instructed. There seems to be some error popping up on the console while loading the UML page. Wondering if that's already been reported?

File: src/app/components/left-sidebar/left-sidebar.component.ts [hunk 1]
```diff
@@ -105,6 +106,20 @@ export class LeftSidebarComponent implements OnInit, AfterViewInit {
 
   }
 
+  hideExpandandCollapse() {
+    var routes: String[] = ["/views/monitoring","/views/config","/views/uml","/views/querying/console","/views/querying/relational-algebra"];
+    var check: Boolean[] = [];
```
Comment (datomo) [inline src/app/components/left-sidebar/left-sidebar.component.ts:111]: Please us `const` or `let` here

**Comment by datomo** [comment]
@Marc13000 Thank you very much for your changes. This looks good. If you adjust my minor comment we should be good to go.

**Comment by Marc13000** [comment]
Done!

**Review by datomo — APPROVED** [review]
This looks fine now, thanks @Marc13000 for this PR.

**Pull request merged**